lib/ecto/type.ex

```diff
@@ -650,3 +697,9 @@ defmodule Ecto.Type do
     {:ok, Enum.reverse(acc)}
   end
 end
+
+if Code.ensure_loaded?(Poison) do
+  defimpl Poison.Encoder, for: Decimal do
```

Comment:
I had no idea where to put this...
Should we make a separate file with all the encoders?